### PR TITLE
Fixed a bug in which regular user has no access to contests

### PR DESCRIPTION
**Pull request by PetarNeshkov**
Closes: https://github.com/SoftUni-Internal/exam-systems-issues/issues/609

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx [hunk 1]
```diff
@@ -39,33 +43,79 @@ const ProblemSubmissions = () => {
...
+            </div>
+        ),
+        [],
+    );
+
+    const renderErrorMessage = useCallback(
+        () => {
+            if (!isNil(problemSubmissionsError)) {
+                const { detail } = problemSubmissionsError;
+                return renderErrorHeading(detail);
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx:64]: This should not be a heading. Make it a regular paragraph.

**Review by shsimeonova — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx (hunk 1 above)
Comment (shsimeonova) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx:61]: Rename functions

**Review by shsimeonova — CHANGES_REQUESTED** [review]

**Review by mishogv — DISMISSED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.module.scss [hunk 2]
```diff
@@ -14,6 +15,16 @@
   }
 }
 
+.problemSubmissionsError {
+  @extend %contentSize;
+
+  color: colors.$text-dark-gray-color;
+  font-family: Lato, serif;
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.module.scss:22]: extend from font mixins

**Review by shsimeonova — CHANGES_REQUESTED** [review]

**Review by mishogv — DISMISSED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx [hunk 3]
```diff
@@ -39,33 +43,76 @@ const ProblemSubmissions = () => {
...
+            </p>
+        ),
+        [],
+    );
+
+    const renderErrorMessage = useCallback(
+        () => {
+            if (!isNil(problemSubmissionsError)) {
+                const { detail } = problemSubmissionsError;
+                return renderErrorParagraph(detail);
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/ProblemSubmissions.tsx:61]: No point in splitting this in 2 render functions

File: Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationMessages.cs [hunk 4]
```diff
@@ -54,7 +54,7 @@ public static class Problem
 
         public const string ProblemNotAssignedToUser = "The problem is not part of your problems for the exam.";
 
-        public const string ProblemResultsNotAvailable = "You cannot view the results for this problem.";
+        public const string ProblemSubmissionsNotAvailable = "You cannot view the submissions results for this problem.";
```
Comment (shsimeonova) [inline Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationMessages.cs:57]: submission*

**Review by shsimeonova — CHANGES_REQUESTED** [review]

**Review by shsimeonova — APPROVED** [review]

**Review by mishogv — APPROVED** [review]

**Pull request merged**